pkg/grpcserver/device.go

```diff
@@ -0,0 +1,183 @@
+package grpcserver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	dmiapi "github.com/kubeedge/mappers-go/pkg/apis/dmi/v1alpha1"
```

Comment:
we can import the DMI in kubeedge 1.12 directly

Reply:
done